### DISCUSSION TITLE
Add OpenSeaMap overlay !

### DISCUSSION THREAD

**Discussion by thesailor** [Leaflet/Leaflet#7876] [Q&A] [answered]
Hey leaflet team

i'm playing with the WP.com plugin for the last few hours. It appears that there is no "overlay" possibility for OSM maps. I'd like to ass the open sea marks overlay on top of a OSM/Leaflet map

https://wiki.openstreetmap.org/wiki/Tiles  + http://tiles.openseamap.org/seamark/${z}/${x}/${y}.png
Sea marks only, background transparent

Any trick? Adding on the feature roadmap? 

thanks a lot !

**Comment by mourner** [comment, marked as answer]
Leaflet is perfectly capable of showing multiple tile layers at once. This might be more of a question to the WP plugin?

**Reply by jonkoops** [reply]
Yeah, this sounds a lot more like a specific issue for the author of the respective Wordpress plugin.
